mongomock/aggregate.py

```diff
@@ -601,6 +601,29 @@ def fixed_getter(doc):
     return fixed_getter
 
 
+def _get_local_field_val(dct, path):
```

Comment:
Please look at `helpers.get_value_by_dot` and consider using it. Also please add tests for some corner cases covered by `get_value_by_dot` not covered by your code here.

Reply:
changed to get_value_by_dot
done